rmf_demo_panel/README.md

```diff
@@ -2,10 +2,9 @@
 
 Setup `rmf_demo_panel`
 
-Install npm, Flask and other dependencies
+Install flask-socketio as the system version is too old
```

Comment:
```suggestion
rosdep will automatically install system version of python3-flask and python3-flask-cors. Yet we will download flask-socketio (5.x) separately via pip since debian version is too old.
```